README.md

```diff
@@ -1471,6 +1472,21 @@ One consequence of this is that if you have an empty string in your `enum` array
 
 If you want to have the field set to a default value when empty you can provide a `ui:emptyValue` field in the `uiSchema` object.
 
+## Localize default messages
+
+You can change localize to default messages errors.
```

Comment:
`You can change localize to default messages errors.`

=> 

`You can change the localization of the error messages.`